python/kfserving/kfserving/storage.py

```diff
@@ -50,12 +45,10 @@ def download(uri: str, out_dir: str = None) -> str:
             Storage._download_s3(uri, out_dir)
         elif re.search(_BLOB_RE, uri):
             Storage._download_blob(uri, out_dir)
-        elif is_local:
-            return Storage._download_local(uri, out_dir)
         else:
             raise Exception("Cannot recognize storage type for " + uri +
-                            "\n'%s', '%s', and '%s' are the current available storage type." %
-                            (_GCS_PREFIX, _S3_PREFIX, _LOCAL_PREFIX))
+                            "\n'%s', '%s' are the current available storage type." %
```

Comment:
Shall update the message to handle the issue #620? The problem is if the path is not exist, will raise exception that `Cannot recognize storage type for ...`, add message such as "If you used PVC:// format, ensure the path exists in the PVC."

Reply:
And we should add "PVC://" here to avoid mislead user.

Reply:
The message to handle the issue #620 is this part:
```
        if not os.path.exists(local_path):
            raise RuntimeError("Local path %s does not exist." % (uri))
```
Notice we cannot add "PVC://" here because in python here we do not support this prefix. The prefix is removed in the injector and replaced with just local path.

Reply:
Seems the part is in `_download_local`?  the function will not be called if not path. For example user mount PVC as "/mnt/pvc" and model is `./export`, but in fact the `./export` does not exist...

Reply:
Yes, you are right. I will update this part.